packages/create-turbo/src/cli.ts

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+
+import chalk from "chalk";
+import { Command } from "commander";
+import notifyUpdate from "./utils/notifyUpdate";
+import { turboGradient } from "./logger";
+
+import { create } from "./commands";
+import cliPkg from "../package.json";
+
+const createTurboCli = new Command();
+
+// create
+createTurboCli
+  .name(chalk.bold(turboGradient("create-turbo")))
+  .description("Create a new Turborepo")
+  .usage(`${chalk.bold("<project-directory> <package-manager>")} [options]`)
+  .argument("[project-directory]")
+  .argument("[package-manager]")
+  .option(
+    "--skip-install",
+    "Do not run a package manager install after conversion",
+    false
+  )
+  .option(
+    "-e, --example [name]|[github-url]",
+    `
+  An example to bootstrap the app with. You can use an example name
+  from the official Turborepo repo or a GitHub URL. The URL can use
+  any branch and/or subdirectory
+`
+  )
+  .option(
+    "--example-path <path-to-example>",
```

Comment:
any short alias for this? maybe `-p`?

Reply:
I didn't add one so we would match `create-next-app` exactly, but not opposed to an alias here.